schemas/openshift/terraform-output-format-1.yml

```diff
@@ -0,0 +1,43 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /openshift/terraform-output-format-1.yml
+  provider:
+    type: string
```

Comment:
Can you add a `description` that clarifies that this is actually the format? I understand the use of the `provider` name so that it is understood that we're using the provider pattern.

Reply:
[d0c3733](https://github.com/app-sre/qontract-schemas/pull/119/commits/d0c37333c1a7411e7156a4d0c9c39aa2d5f8b6ab)